phpcs.xml.dist

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<ruleset name="SilverStripe">
+    <description>CodeSniffer ruleset for SilverStripe coding conventions.</description>
+
+    <file>src</file>
+
+    <rule ref="PSR12" >
+        <exclude name="PSR1.Methods.CamelCapsMethodName.NotCamelCaps" />
+        <exclude name="PSR1.Files.SideEffects.FoundWithSymbols" />
```

Comment:
Are either of these exceptions necessary for this project? If not, remove them.

Reply:
Updated